orchprovider/k8s/v1/k8s.go

```diff
@@ -1100,7 +1165,7 @@ func (k *k8sOrchestrator) getReplicaPods(vsm string, podOps k8sCoreV1.PodInterfa
 	return rp, nil
 }
 
-// getPods deletes the Pods w.r.t the VSM
+// getPods gets the Pods w.r.t the VSM
```

Comment:
same as above (Volume)

Reply:
Ok. These will be refactored to volume_add.go & so on files. Will be doing these renaming stuff if it exists after all these refactorings.